frontend/src/routes/(authenticated)/user/+page.svelte

```diff
@@ -73,11 +71,10 @@
     }
   });
 
-  // This is a bit of a hack to make sure that the email field is not required if the user has no email
-  // even if the user edited the email field
-  run(() => {
-    if (!$form.email && $user && !$user.email) $form.email = null;
-  });
+  // TODO: We used to have a hack here to make sure that the email field is not required if the user
```

Comment:
I made the same suggestion elsewhere. If this works, that would be kind of nice:
```
$effect(() => {
    if (!$form.email && $user && !$user.email && $form.email !== null) $form.email = null;
  });
```